packages/commercetools/composables/src/useStore/factoryParams.ts

```diff
@@ -10,7 +10,7 @@ async function load (context: Context, params: UseStoreFactoryLoadParamArguments
 
 // Change param
 async function change (context: Context, { store }: UseStoreFactoryChangeParamArguments) {
-  context.$ct.config.storeService.changeCurrentStore(store.id);
+  context.$ct.config.storeService.changeCurrentStore(`${store.key}`);
```

Comment:
ok you are setting the current store in the config... but then you are reloading the page... so what will happen? i think the default store will be loaded from the config, not the one you just set

Reply:
i don't remember - is there any cookie storing underneath?

Reply:
The `changeCurrentStore` service handle the cookie storage.

Here on line 32:
https://github.com/vuestorefront/vue-storefront/blob/EN-356-implement-useStore-commercetools/packages/commercetools/composables/nuxt/plugin.js